### ISSUE TITLE
Recipe tasks and their generated Put tasks being linked

### ISSUE THREAD

**Issue by CasparPetch** [MortenTobiasNielsen/Factorio-GUI-V2#361]
Something that could save a lot of time - example: changing recipe inserter 5 to recipe inserter 10 automatically updates the 3 put tasks of iron plates, gears, circuits all to 10. Would save a lot of manual number typing. Not sure if feasible or not :)

**Comment by MortenTobiasNielsen** [comment]
I have thought about adding this, but I could't figure out a good way of ensuring that it wouldn't do something the user wouldn't expect. Lets say that you have added a few steps in between the recipe and the put steps, due to it being more optimal to do that, should it then search until it finds the according steps, how far should it search, what if 2 out of 3 put steps can be found, should it then scale the 2 steps, should it add the third? 

The way I "solve" the issue is to double click the recipe step, set it to 10, add the step and then delete the x5 recipe with its put steps. This can be done in about 10 seconds. 🙂

**Comment by MortenTobiasNielsen** [comment]
Another way to do it if you have a large amount of recipe steps you need to change would be to use the template and multiply with two in your example.

**Comment by theis999** [comment]
I have been thinking of this type of feature. I expect it to be **possible** to implement, however it will **not** be **trivial**.
So i will put this on-hold, at least for now.

The obvious question for implementation, is whether they should be linked explicitly or implicitly?

**Comment by MortenTobiasNielsen** [comment]
How do you define implicit and explicit in this context?

**Comment by CasparPetch** [comment]
> The way I "solve" the issue is to double click the recipe step, set it to 10, add the step and then delete the x5 recipe with its put steps. This can be done in about 10 seconds. 🙂

Yeah that's the solution I use as well but was just thinking of ways it could be slightly less tedious. Inplace editor (which I got to try yesterday) already saves most of the lost time anyway so I'm not too worried either way

**Comment by theis999** [comment]
Implicit would likely mean that adjacent steps could be considered linked. For instance if there is a set-recipe step followed by put actions into the same entity.

With explicit meaning there would be column for linking steps together. (we would have to scrap multi-build to make room)

**Comment by MortenTobiasNielsen** [comment]
Okay. That was what I expected, but it can be good to clarify. 🙂 I don't see a good way to do this, at least with the way I used it when I actively used the tool.

**Comment by theis999** [comment]
I agree that it won't be easy, and likely have little value too.
